README.adoc

```diff
@@ -9,7 +9,7 @@ Dan Allen <https://github.com/mojavelinux[@mojavelinux]>; Guillaume Grossetie <h
 ifndef::uri-rel-file-base[:uri-rel-file-base: link:]
 :uri-contribute: {uri-rel-file-base}CONTRIBUTING.adoc
 :uri-contribute-code: {uri-rel-file-base}CONTRIBUTING-CODE.adoc
-:uri-user-manual: https://asciidoctor-docs.netlify.com/asciidoctor.js/3.0
+:uri-user-manual: https://asciidoctor-docs.netlify.com/asciidoctor.js/latest
```

Comment:
Thanks!
Now that the new documentation site is live, we should use: 

```suggestion
:uri-user-manual: https://docs.asciidoctor.org/asciidoctor.js/latest/
```

Reply:
Oh, thanks. My commit is evolved 😄